crates/uv/tests/pip_show.rs

```diff
@@ -56,6 +56,126 @@ fn show_empty() {
     );
 }
 
+#[test]
+fn show_requires_multiple() -> Result<()> {
+    let context = TestContext::new("3.12");
+
+    let requirements_txt = context.temp_dir.child("requirements.txt");
+    requirements_txt.touch()?;
+    requirements_txt.write_str("requests==2.31.0")?;
+
+    uv_snapshot!(install_command(&context)
+        .arg("-r")
+        .arg("requirements.txt")
+        .arg("--strict"), @r###"
+    success: true
+    exit_code: 0
+    ----- stdout -----
+
+    ----- stderr -----
+    Resolved 5 packages in [TIME]
+    Downloaded 5 packages in [TIME]
+    Installed 5 packages in [TIME]
+     + certifi==2023.11.17
+     + charset-normalizer==3.3.2
+     + idna==3.4
+     + requests==2.31.0
+     + urllib3==2.1.0
+    "###
+    );
+
+    context.assert_command("import requests").success();
+    let filters = [(
+        r"Location:.*site-packages",
+        "Location: [WORKSPACE_DIR]/site-packages",
+    )]
+    .to_vec();
+
+    // Guards against the package names being sorted.
+    uv_snapshot!(filters, Command::new(get_bin())
+        .arg("pip")
+        .arg("show")
+        .arg("requests")
+        .arg("--cache-dir")
+        .arg(context.cache_dir.path())
+        .env("VIRTUAL_ENV", context.venv.as_os_str())
+        .current_dir(&context.temp_dir), @r###"
+    success: true
+    exit_code: 0
+    ----- stdout -----
+    Name: requests
+    Version: 2.31.0
+    Location: [WORKSPACE_DIR]/site-packages
+    Requires: certifi, charset-normalizer, idna, urllib3
+
+    ----- stderr -----
+    "###
+    );
+
+    Ok(())
+}
+
+#[test]
+#[cfg(not(windows))]
```

Comment:
not super clear why windows doesn't pull `tzdata` https://github.com/astral-sh/uv/actions/runs/8225296455/job/22490152644, may continue to look while having this open for review 😭

Reply:
We have a windows filter hack in the snapshots for our common test packages: https://github.com/astral-sh/uv/blob/1181aa9be40b7f99334f7efd15d5102653d8b38b/crates/uv/tests/common/mod.rs#L370-L376

Could you use a pure python package for the test case? Pandas and numpy wheel are big due to their native modules (something like 5-20 MB) and since we run the tests with no cache these add up.

Reply:
updated to use `click`.
```
$ du -sh /Users/chan.kang/test/uv/venv/lib/python3.12/site-packages/click
824K	/Users/chan.kang/test/uv/venv/lib/python3.12/site-packages/click
```